tools/openshift-ci/Dockerfile

```diff
@@ -5,6 +5,9 @@ LABEL maintainer="OCP QE Team"
 
 USER root
 
+RUN sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-Linux-* &&\
```

Comment:
@dis016 Minor formatting issue - Please add a space between && and \ .
FYI, I have tested the Dockerfile on my RHEL and able to create image successfully without original reported issue.